testing/rust/minimize-rpath.patch

```diff
@@ -0,0 +1,93 @@
+--- a/src/librustc_trans/back/rpath.rs
```

Comment:
Please add patch header and description – what this patch actually do and why it’s needed.